wrappers/python/simtk/openmm/app/amberprmtopfile.py

```diff
@@ -192,6 +202,11 @@ def createSystem(self, nonbondedMethod=ff.NoCutoff, nonbondedCutoff=1.0*unit.nan
             total mass the same.
         ewaldErrorTolerance : float=0.0005
             The error tolerance to use if nonbondedMethod is Ewald or PME.
+        switchDistance : float=0*nanometers
+            A positive value turns on a potential energy switching function for
+            Lennard-Jones interactions. If the switchDistance is 0 or evaluates
+            to boolean False, no switching function will be used. Illegal values
+            (e.g., less than 0) will raise a ValueError
```

Comment:
This description doesn't say what `switchDistance` *is*.  It makes it sound like just a boolean flag, where all that matters is the sign.